src/doc_builder/commands/build.py

```diff
@@ -51,6 +107,47 @@ def build_command(args):
     if "pr_" not in version and os.path.isfile(os.path.join(package_doc_path, "_versions.yml")):
         update_versions_file(os.path.join(args.build_dir, args.library_name), version)
 
+    # If asked, convert the MDX files into HTML files.
+    if args.html:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            tmp_dir = Path(tmp_dir)
+            # Copy everything in a tmp dir
+            shutil.copytree(kit_folder, tmp_dir / "kit")
+            if (tmp_dir / "kit" / "src" / "routes" ).is_dir():
+                # Remove the current MDX dir if it exists.
+                shutil.rmtree(tmp_dir / "kit" / "src" / "routes")
```

Comment:
Is it possible to only remove the .mdx files?

For now there is only a `.gitkeep` file in `kit/src/routes`, but with the viewer PR there will be a `__layout.svelte` and an `endpoints` folder (containing a `.ts` file) that should stay